datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1302,6 +1303,26 @@ impl LogicalPlan {
         Wrapper(self)
     }
 
+    /// Return a `format`able structure that produces plan in postgresql JSON format.
```

Comment:
Likewise here it would be nice to add a link to the spec

Reply:
See comments above.